src/lib/chat/matrix-client.ts

```diff
@@ -159,6 +159,46 @@ export class MatrixClient implements IChatClient {
     return admins;
   }
 
+  /**
+   * Currently we have minimum invite, kick level set to 100 for all users
+   * this means that only the room creator/admin (which has a power_level of 100) can invite and kick users
+   *
+   * this function will lower the minimum invite and kick levels to 50 for all rooms,
```

Comment:
Suggestion: Make sure you have a task set for a week(?) from now to remove this. Then, you can just adjust the room when the admin goes to add new moderators if it isn't yet set up for it.

Reply:
noted, thanks